### PR TITLE
Remove x64 check from installer

### PR DESCRIPTION
**Pull request by tyrielv**
PR #1815 updated the installer dependency version, which seems to have caused the installer to enforce the AllowedArchitectures=x64 check (which has been there for years). We've received complaints from users that the latest release refuses to install on their ARM machines, but previous versions did and worked fine with the default x86/64 emulation.

Although ARM is not explicitly supported, we should not prevent users from installing GVFS on ARM when it has been working for them so far.

### CODE REVIEW

**Review by derrickstolee — APPROVED** [review]

File: GVFS/GVFS.Installers/GVFS.Installers.csproj [hunk 1]
```diff
@@ -28,9 +28,7 @@
     <Content Include="$(PkgGitForWindows_GVFS_Portable)\tools\*" />
     <Content Include="install.bat" />
     <Content Include="info.bat" />
-    <NuspecTemplate Include="GVFS.Installers.template.nuspec"
```
Comment (mjcheetham) [inline GVFS/GVFS.Installers/GVFS.Installers.csproj:31]: Please can you revert the other changes to this file (not the `Tools.InnoSetup` version bump of course)?

**Review by mjcheetham — CHANGES_REQUESTED** [review]
Looks good, just need to revert the `&quot;` that got added to the csproj file by Visual Studio. Thanks!

File: GVFS/GVFS.Installers/GVFS.Installers.csproj (hunk 1 above)
Comment (tyrielv) [reply GVFS/GVFS.Installers/GVFS.Installers.csproj]: Done

**Review by mjcheetham — APPROVED** [review]
Thank you @tyrielv!

**Pull request merged**